ios/PresentationLayer/Onboarding/Sources/Onboarding/Login/LoginView.swift

```diff
@@ -27,12 +27,12 @@ struct LoginView: View {
             Spacer()
             PrimaryAndSecondaryButtons(
                 primaryButtonTitle: L10n.login_view_login_button_title,
-                primaryButtonLoading: viewModel.state.loginButtonLoading,
                 onPrimaryButtonTap: { viewModel.onIntent(.login) },
                 secondaryButtonTitle: L10n.login_view_register_button_title,
                 onSecondaryButtonTap: { viewModel.onIntent(.register) }
             )
         }
+        .environment(\.isLoading, viewModel.state.loginButtonLoading)
```

Comment:
Maybe change `viewModel.state.loginButtonLoading` to just `viewModel.state.loading`? As now it is used in context of the whole view, not just in context of the login button.

Reply:
Done